cmd/nem/main.go

```diff
@@ -265,7 +317,15 @@ func timeInSeconds(val float64) string {
 
 // printAccountDetails prints account details in pretty user-oriented multi-line format
 func printAccountDetails(chain core.Chain, pair keypair.KeyPair) {
+	printAddress(chain, pair)
+	printPublicKey(pair)
+	fmt.Println("Private key:", hex.EncodeToString(pair.Private))
```

Comment:
You extracted `public address` and `public key`, is `private key` an exception from this rule?